packages/backend/src/services/transaction-request-service.ts

```diff
@@ -133,36 +117,89 @@ export class KnexTransactionRequestService {
     this._knex = knex
   }
 
-  async create (transactionRequest: TransactionRequestsPostRequest, userId: number): Promise<StoredRequest> {
-    const transactionRequestTools = new TransactionRequestTools(transactionRequest)
+  async create (transactionRequest: TransactionRequestsPostRequest, userId: number): Promise<TransactionRequestProps> {
+    if (isValid(transactionRequest).error) {
+      throw new Error(isValid(transactionRequest).error.message)
```

Comment:
You are validating the object twice just to get the error message. Move the isValid to above and store the result before doing the if check